core/SConscript.unix

```diff
@@ -117,6 +117,7 @@ if FEATURE_FLAGS["SECP256K1_ZKP"]:
     CPPDEFINES_MOD += [
         'USE_SECP256K1_ZKP',
         'USE_SECP256K1_ZKP_ECDSA',
+        'USE_SECP256K1_ZKP_BIP340',
```

Comment:
Is this used anywhere? I might have missed that

Reply:
As far as I know, this line is not needed. The symbol `USE_SECP256K1_ZKP_BIP340` isn't used anywhere.

Reply:
I can see it being used in `core/embed/extmod/modtrezorcrypto/modtrezorcrypto.c` (lines 46 and 95).

Reply:
Because we haven't used the python module `bip340` yet.

Reply:
I hadn't run the tests on hardware. It turns out this branch does not work on the device. There is nothing to stop the build from succeeding. It fails at runtime because the module does not import.

I don't think it makes sense to have this compile flag, because the code can't work without it and we have no alternative. Same applies to https://github.com/trezor/trezor-firmware/blob/master/core/src/trezor/crypto/curve.py#L3-L6.

Reply:
https://github.com/trezor/trezor-firmware/pull/1891/commits/a0368f504ba883463baa6f698781ecb34afe0ccb